src/Symfony/Component/Routing/RequestContext.php

```diff
@@ -45,6 +45,23 @@ public function __construct(string $baseUrl = '', string $method = 'GET', string
         $this->setQueryString($queryString);
     }
 
+    public static function fromBaseUri(string $baseUri, string $host = 'localhost', string $scheme = 'http', int $httpPort = 80, int $httpsPort = 443): self
```

Comment:
@nicolas-grekas: maybe update the name of the function and some variable here too?

Reply:
Not sure, why would this be named default? It's just a factory.

Reply:
Did you mean something else? Did you have something in mind?

Reply:
I was considering to rename this method to “fromDefaultUri”. But I’m not convinced it is a better name just because it reflects on the config parameter.

Reply:
Why not just `fromUri()` then?

Reply:
updated